.gitignore

```diff
@@ -1,2 +1,3 @@
 /.bundle
 /native-helpers
+git.store
```

Comment:
What is `git.store`?

Reply:
I was running this locally on my machine and it looks like dependabot stores git credentials in that file

Reply:
Ok 👍